### DISCUSSION TITLE
Has pull #2016 made osm2pgsql slightly faster?

### DISCUSSION THREAD

**Discussion by mboeringa** [osm2pgsql-dev/osm2pgsql#2025] [Q&A] [answered]
Just a minor question. I am currently re-running a Planet **non-slim** import, and noticed it is slightly faster (by a few percentages) than all previous non-slim runs, for both nodes, ways and relation loading (e.g. 75k/s way loading versus 70-72k/s). 

Now the only significant change is that I recompiled osm2pgsql with https://github.com/openstreetmap/osm2pgsql/pull/2016 included (latest master in fact). At least, as far as I can judge the list of merged pull requests...

Is it possible that this pull made osm2pgsql slightly faster? (of course a good thing ;-))

I am asking as I think it unlikely at this point to be just a lucky anomaly with this session. The last compiled state I am comparing this new result to is osm2pgsql version 1.8.1 (1.8.1-183-g5dba2405).

**Comment by joto** [comment, marked as answer]
> Is it possible that this pull made osm2pgsql slightly faster?

No

**Reply by mboeringa** [reply]
OK, besides the option of this still being a lucky quick run, maybe a recent Windows update (which is the host system), or a Ubuntu Linux kernel update (guest system), made it slightly faster. Nothing else changed at least.
